certification/pyxis/pyxis.go

```diff
@@ -127,6 +127,44 @@ func (p *pyxisClient) getImage(ctx context.Context, dockerImageDigest string) (*
 	return &data.Data[0], nil
 }
 
+// updateImage updates a given certification image based on how the image is built in the `submit` flow
+func (p *pyxisClient) updateImage(ctx context.Context, certImage *CertImage) (*CertImage, error) {
+	b, err := json.Marshal(certImage)
```

Comment:
I still think you should create a new CertImage to pass to the PATCH call. Passing in the *CertImage was just to keep the signature from having to grow if we ever needed to add another field.

Reply:
I initially had the struct to post in the updateImage method, but moved it out since I thought that's what we were going for, and was probably confused by the ask. I moved it back here.